### PR TITLE
change to new button V2

### PR DESCRIPTION
**Pull request by asaadam**
change to new button V2

### CODE REVIEW

File: uikit/buttonV2/Button.tsx [hunk 1]
```diff
@@ -36,13 +39,13 @@ const buttonClasses = {
     variant: {
         primary: "bg-dark-neutral-primary text-dark-background-default hover:bg-dark-neutral-soft",
         secondary: "bg-light-neutral-subtle/10 text-dark-neutral-primary hover:bg-light-neutral-subtle/20",
-        alternate: "bg-dark-primary text-white hover:bg-dark-primary-soft",
+        alternate: "bg-dark-primary text-white hover:bg-light-darker",
         discord: "bg-discord-dark text-white hover:bg-discord-light",
         twitter: "bg-twitter-dark text-white hover:bg-twitter-light",
```
Comment (math-marcellino) [inline uikit/buttonV2/Button.tsx:44]: The bg color for variant `discord` and `twitter` is reversed between the default and hover state sir (default should be light, hover should be dark)
image

File: modules/homePage/component/v2/FourthJumbotron/FourthJumbotronComponent.tsx [hunk 2]
```diff
@@ -17,8 +18,12 @@ function Image({ src, alt, className }: { src: string; alt: string; className?:
     );
 }
 
-function Button({ children, className }: { children?: React.ReactNode; className?: string }) {
-    return <ButtonPrimary className={`absolute bottom-[24px] left-[24px] text-gray-100 ${className || ""}`}>{children}</ButtonPrimary>;
+function ButtonJumbotron({ children, className, variant = "primary" }: { variant?: ButtonVariant; children?: React.ReactNode; className?: string }) {
```
Comment (math-marcellino) [inline modules/homePage/component/v2/FourthJumbotron/FourthJumbotronComponent.tsx:21]: I think the `classname` here is not necessary anymore sirr, but CMIIW

**Review by math-marcellino — CHANGES_REQUESTED** [review]
Here are some minor issues that I found sir, everything else LGTM!

File: uikit/buttonV2/Button.tsx (hunk 1 above)
Comment (asaadam) [reply uikit/buttonV2/Button.tsx:44]: :( :( :( :( :( :( :( :( :(

**Pull request closed without merging**